src/Proxy/FilterPathProxy.php

```diff
@@ -17,7 +16,7 @@
 use ValueObjects\Web\Domain;
 use ValueObjects\Web\PortNumber;
 
-class CalendarSummaryProxy extends Proxy
+class FilterPathProxy extends Proxy
```

Comment:
Hardcoden op `GET` is idd oké. Maar het is niet een default maar een "constante" (bij wijze van spreken), want je kan het niet aanpassen. Daarom mijn voorstel om het oftewel configureerbaar te maken (al dan niet met `GET` als default), of om de naam aan te passen zodat het duidelijk is dat die filter enkel voor `GET` requests werkt.